forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -0,0 +1,326 @@
+                                                if let Ok(height) = height.as_str().parse::<i64>() {
+                                                    if date > latest_date {
+                                                        latest_date = date;
+                                                        latest_height = height;
+                                                        snapshot_to_keep = Some(path.clone());
+                                                    } else if date == latest_date
+                                                        && height > latest_height
+                                                    {
+                                                        latest_height = height;
+                                                        snapshot_to_keep = Some(path.clone());
+                                                    }
+
+                                                    snapshots_with_valid_name.push(path);
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+
+                let term = Term::stdout();
+                if snapshots_with_valid_name.len() < 2 {
+                    term.write_line("No files to delete").unwrap();
+                    return;
+                }
+
+                let mut snapshots_to_delete = snapshots_with_valid_name;
+                let mut index_to_keep = 0;
+                term.write_line("Files to delete:").unwrap();
+                if let Some(snapshot_to_keep) = snapshot_to_keep {
+                    for (i, path) in snapshots_to_delete.iter().enumerate() {
+                        if &snapshot_to_keep != path {
+                            term.write_line(&format!("{:?}", path.as_path())).unwrap();
+                        } else {
+                            index_to_keep = i;
+                        }
+                    }
+                }
+                snapshots_to_delete.remove(index_to_keep);
+
+                if !yes
+                    && !Confirm::new()
+                        .with_prompt("Do you want to continue?")
+                        .interact()
+                        .unwrap()
+                {
+                    term.write_line("Aborted.").unwrap();
+                    return;
+                }
+
+                for snapshot_path in snapshots_to_delete {
+                    delete_snapshot(&snapshot_path);
+                }
+            }
+        }
+    }
+}
+
+fn default_snapshot_dir(config: &Config) -> PathBuf {
+    config
+        .client
+        .data_dir
+        .join("snapshots")
+        .join(config.chain.name.clone())
+}
+
+fn delete_snapshot(snapshot_path: &PathBuf) {
+    let term = Term::stdout();
```

Comment:
Why do you use `term.write_line` and not just regular print macros?

Reply:
fixed